class LocationAlreadyExistsError(Exception):
    """Exception raised when a location with the same name already exists."""

    pass
